doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -132,6 +132,13 @@ Matter
   * Page about :ref:`ug_matter_device_optimizing_memory`.
   * Shell commands for printing and resetting the peak usage of critical system resources used by Matter.
     These shell commands are available when both :kconfig:option:`CHIP_LIB_SHELL` and :kconfig:option:`CHIP_STATISTICS` Kconfig options are set.
+  * Reaction to the last fabric removal.
+    Now the user can decide what should happen after removing the last fabric:
+    
+    * Do nothing (:kconfig:option:`CHIP_LAST_FABRIC_REMOVED_NONE`).
+    * Erase the saved network credentials (Thread or WI-FI) (:kconfig:option:`CHIP_LAST_FABRIC_REMOVED_ERASE_ONLY`).
```

Comment:
I think we erase more than just network credentials, right? Basically we erase the whole NVS storage. Could we change it, as the current sentence may suggest to customer that only network credentials will be lost and the rest will be kept.